importer/loader.py

```diff
@@ -57,13 +64,22 @@ def load(self):
             library_name = self.__extract_float_value(i, library_name_column)
             if library_name is None:
                 library_name = sample_name
-            reads.append(RawRead(
-                self.__extract_text_value(i, filename_column),
-                self.__extract_text_value(i, mate_filename_column),
-                sample_name,
-                self.__extract_text_value(i, sample_accession_column),
-                self.__extract_float_value(i, taxon_id_column),
-                library_name))
+            if filename_column is not None:
+                reads.append(RawRead(
+                    self.__extract_text_value(i, filename_column),
+                    self.__extract_text_value(i, mate_filename_column),
+                    sample_name,
+                    self.__extract_text_value(i, sample_accession_column),
+                    self.__extract_float_value(i, taxon_id_column),
+                    library_name))
+            if run_accession_column is not None:
+                reads.append(RawRead(
```

Comment:
Not very clean to use the same class and "redefine" the meaning of the individual attributes.